docs/glossary.md

```diff
@@ -0,0 +1,194 @@
+
+### Validity Proof
+A type of cryptographic proof that can be used to attest to the validity of a state transition. For example, zk-Rollups utilize validity proofs to prove valid state transitions to a parent chain - commonly used with proof systems such as SNARKs and STARKs.
+
+## Data Management and Availability
+### Data Availability Sampling (DAS)
+Data availability sampling allows light nodes to confirm the availability of data without downloading the complete block. Through this method, light nodes engage in several rounds of random sampling for small chunks of block data. With each successful round, confidence that the data is available grows. When the light node achieves a set confidence threshold, they recognize the block data as accessible.
+In Avail's blockchain architecture, this sampling technique empowers light nodes to enhance the network's security and capacity, all while using much less resource-intensive hardware compared to full nodes.
+
+### KZG Commitments
+KZG commitments, introduced by Aniket Kate, Gregory M. Zaverucha, and Ian Goldberg in 2010, provide a way to commit to polynomials in a succinct manner. Recently, polynomial commitments came to the forefront, being primarily used as commitments in PLONK-like zero knowledge constructions.
+
+In Avail’s construction, we use KZG commitments for the following reasons:
+
+* It allows us to commit to values in a succinct manner to be kept inside the block header.
+* Short openings are possible which helps a light client verify availability.
+* The cryptographic binding property helps us avoid fraud proofs by making it computationally infeasible to produce incorrect commitments.
+
+
+
+### Erasure Coding
+
+It is a method of reproducing data from smaller encoded blocks. The light client recovers application data from the already erasure-encoded blocks on Avail. It’s important because the LC is sampling randomly, so block-level redundancy makes it more likely that the LC would find missing data.
+
+
+### Fraud Proofs
+Fraud proofs are a type of cryptographic proof that can be used to prove that a transaction or state transition is invalid. Fraud proofs are particularly useful for blockchain applications, where it is important to be able to verify the validity of transactions and state transitions quickly and efficiently.
+
+### Data Availability Committee (DAC)
+A Data Availability Committee (DAC) is a trusted group of data managers (nodes) that store and grant access to data, maintaining the data off the main chain. Upon receiving and verifying data requests, the DAC provides authorized users with the needed data.
```

Comment:
Idea here should be to also highlight Avail as a DA layer as opposed to a DAC / trusted data managers providing data availability.